pyproject.toml

```diff
@@ -78,6 +64,16 @@ requires-python = ">=3.9,<4.0"
 [project.optional-dependencies]
 burr = ["burr[start]==0.22.1"]
 docs = ["sphinx==6.0", "furo==2024.5.6"]
+other = [
```

Comment:
split them in three groups of extras, one for "other" language models, one for more semantic options, one for more browser options

Reply:
ok @DiTo97, I updated